### ISSUE TITLE
Using RS232 to TTL adapter

### ISSUE THREAD

**Issue by ssshake** [Lameguy64/ps2serial#1]
Hi hope you dont mind me asking this question here. I was wondering if your code could be used with this RS232 to TTL adapter. I ask because I have a box of them, and I've used them with other vintage pc projects (building wifi modems).

I'm not familiar with the components you list in your readme. I'm wondering if those components and this adapter serve the same purpose.

Thanks!

**Comment by corvus1** [comment]
yes, you can absolutely use a max232, however, it might require a small change in software for RTS sensing.

**Comment by Lameguy64** [comment]
It bugs me people are still using this clumsy implementation of mine over my successor of this, but then the fact it is written in AVR assembly is likely a deterrent to most from adopting it.

**Comment by corvus1** [comment]
@Lameguy64 Any links to the successor implementation? 
Although, I gotta say, it might be clumsy, but it's easy to understand and change if need be.

**Comment by corvus1** [comment]
So, I found your asm implementation, and it's cool and everything, but I for one really don't want to dig into AVR assembly. If it's performance you're after, you can get a RasPi Pico or a bluepill clone from aliexpress for the price of an arduino nano, which is about 3 bucks. And it will have a proper USB interface, which will allow you to use any old usb mouse, and it has 20 times the performance, while the same power consumption, and you can program in micro python if you so choose, which is a lot easier to understand. So why all the assembly?

What bugs me really is that ttl-rs232 coverter boards from ali only connect 1 line in and 1 line out. So you can either have the RTS sense, or software configuration. Despite max232 having 2 lines in each direction, they just aren't connected.

**Comment by PPEyt** [comment]
You can also use a NPN Transistor and 2 resistors instead of the IC, i also did this because i dont have that chip.
